### PR TITLE
Add support for s390x

### PR DESCRIPTION
**Pull request by sangitanalkar**
### What does this PR achieve? Why do we need it?
As Travis CI [officially supports](https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z) s390x builds, adding support for same and also release binaries for s390x.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,14 @@ install:
 - bash .travis-fork-fix
 - go get github.com/mattn/goveralls
 - go get github.com/wadey/gocovmerge
-- curl https://glide.sh/get | sh
+- if [[ $(uname -m) == 's390x' ]]; then
+      wget https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-s390x.tar.gz;
```
Comment (phlogistonjohn) [inline .travis.yml]: Please remove the unnecessary trailing semicolons on this line and the subsequent lines.

File: .travis.yml [hunk 2]
```diff
@@ -9,7 +9,14 @@ install:
...
+      tar -xzf glide-v0.13.1-linux-s390x.tar.gz linux-s390x/glide --strip=1;
+      export PATH=$PWD:$PATH;
+      glide --version;
```
Comment (phlogistonjohn) [inline .travis.yml]: What does this call to `glide --version` accomplish?

File: .travis.yml (hunk 1 above)
Comment (sangitanalkar) [reply .travis.yml]: Hi @phlogistonjohn , removing the trailing semicolons inside `if-else` condition makes the build fail with `syntax error: unexpected end of file`, hence the semicolons are required.

File: .travis.yml (hunk 2 above)
Comment (sangitanalkar) [reply .travis.yml]: `glide --version` is not required here, will remove it and commit the change.

File: .travis.yml (hunk 1 above)
Comment (phlogistonjohn) [reply .travis.yml]: That's weird but may be due to embedding the script within yaml. I'm OK with it for now, perhaps we can improve it later.

**Comment by phlogistonjohn** [comment]
Please squash your changes together. Thanks.

**Comment by sangitanalkar** [comment]
@phlogistonjohn I have squashed the commits, let me know if anything else is needed from my side.

**Comment by phlogistonjohn** [comment]
@raghavendra-talur PTAL

**Pull request merged**